bundle/config/mutator/resolve_variable_references.go

```diff
@@ -37,9 +37,22 @@ type resolveVariableReferences struct {
 	lookupFn    func(dyn.Value, dyn.Path, *bundle.Bundle) (dyn.Value, error)
 	skipFn      func(dyn.Value) bool
 	extraRounds int
+
+	// includeResources allows resolving variables in 'resources', otherwise, they are excluded.
```

Comment:
Does it mean that non-resource are processed twice?

Reply:
Right. I'm not sure that the existing variable resolution code will work if we give it only "resources" because it has to take variable values from somewhere.

Reply:
Actually, they aren't processed twice, because we also set "pattern" in `ResolveVariableReferencesOnlyResources`